docs/reference/docs/reindex.asciidoc

```diff
@@ -118,8 +118,11 @@ POST _reindex
 // CONSOLE
 // TEST[setup:twitter]
 
-By default, version conflicts abort the `_reindex` process, but you can just
-count them by setting `"conflicts": "proceed"` in the request body:
+By default, version conflicts abort the `_reindex` process. The `"conflicts"` request body
+parameter can be used to instruct `_reindex` to proceed with next document on version conflicts.
+Notice that the handling of other types of errors is unaffected by the `"conflicts"` parameter.
```

Comment:
Suggestion:
The handling of other types of errors is unaffected by the `"conflicts"` parameter.


I think that the `Notice that` part is unnecessary and feels weird for this documentation. If you do want emphasis, I would probably go with "It is important to note that the handling of other error types is unaffected by the..."

Reply:
I opted for keeping the emphasis with your modified text. The whole point of this PR was this note, since that was the clarification asked for in the issue. See 7f76ba806de6513e68c3d183e3c01f05e3a484f6